mvc/models/articles.models.js

```diff
@@ -72,3 +72,20 @@ exports.insertComment = async (article_id, author, body) => {
     const insertQuery = format(`INSERT INTO comments (article_id, author, body) VALUES ($1, $2, $3) RETURNING *;`)
     return db.query(insertQuery, formattedComment)
 }
+
+exports.updateVotes = async (article_id, votes) => {
+    if (!article_id || !votes) {
+        return Promise.reject({
+            status: 400,
+            msg: 'Bad request'
+        });
+    }
+
+    await checkExists('articles', 'article_id', article_id);
```

Comment:
Please either use async-await or promise chaining in a particular repo and don't mix the two.